documentation/docs/introduction.md

```diff
@@ -21,45 +21,57 @@ In this guide, we will go through the most important concepts that developers wi
 
 ### Chapter 2: Decentralized Identity
 
-Describes the concept of Decentralized or Self-Sovereign Identities (SSI), how it applies to People, Organizations and Things, and why IOTA is used. 
+Describes the concept of Decentralized or Self-Sovereign Identities (SSI), how it applies to People, Organizations and Things, and why IOTA is used.
 
-### Chapter 3.1: Decentralized Identifiers (DID)
+### Chapter 3: Getting Started
+
+Gives a quick overview of how to install the library and creating your first DID.
+
+### Chapter 4: Concepts
+
+Describes relevant concepts of SSI and how to utilize them in the library.
+### Chapter 4.1: Decentralized Identifiers (DID)
 
 Explains the DID standard from W3C and how to manipulate DID Documents.
 
-### Chapter 3.2: Verifiable Credentials (VC)
+### Chapter 4.2: Verifiable Credentials (VC)
 
 Explains the VC standard from W3C, how to create and revoke VCs, and how to use Verifiable Presentations.
 
-### Chapter 3.3: DID Communications (DID Comm)
+### Chapter 4.3: DID Communications (DID Comm)
 
 This chapter covers the DID Comm standard, which is being developed by the Decentralized Identity Foundation (DIF). It also describes the different messages agents may send each other and what the expected responses may look like.
 
-### Chapter 3.4: Advanced Concepts
+### Chapter 4.4: Advanced Concepts
 
 This chapter is meant for those that want to push the IOTA Identity framework to its limits, utilizing the more complex, yet more flexible lower-level libraries, allowing developers to optimize their implementation, take control over storage/security, and add features to the framework. 
 
-### Chapter 4: Programming Languages
+### Chapter 5: Programming Languages
 
 While the framework itself is developed in the Rust programming language, we also provide bindings, or "Foreign Function Interfaces" (FFI), to other languages. These will have separate getting started sections, making the rest of the guide language agnostic, focusing on the conceptual level. 
 
-### Chapter 5: Specification
+### Chapter 6: Tutorials
+Contains end-to-end examples using the library to achieve common use-cases.
+
+### Chapter 7: Specification
 
 While IOTA Identity implements many existing standards, it also adds some additional features we would like to standardize ourselves. This chapter covers these features and how they work in great detail. These are not light reads and can be skipped. 
 
 
-### Chapter 6: Glossary
+### Chapter 8: Glossary
 
 A list of all terminology used in this guide, the framework, and all materials surrounding it. 
 
-### Chapter 7: Contribute
+### Chapter 9: Contribute
 
 A simple guide on how to contribute to the framework.
 
-### Chapter 8: Contact
+### Chapter 10: Workflow
+A overview of the workflow to design, develop and release the library.
```

Comment:
```suggestion
An overview of the workflow to design, develop and release the library.
```

I don't quite get what this describes, but if this refers to the IOTA Identity framework, then use framework instead of library.